packages/frontend/amp/components/Ad.tsx

```diff
@@ -37,6 +37,36 @@ const adStyle = css`
     }
 `;
 
+const adClass = css`
+    display: none;
+`;
+
+const usAdRegionClass = css`
+    .amp-geo-group-us & {
+        display: block;
+    }
+`;
+
+const auAdRegionClass = css`
+    .amp-geo-group-au & {
+        display: block;
+    }
+`;
+
+const rowAdRegionClass = css`
+    .amp-geo-group-eea & {
```

Comment:
How does that work for Non-EEA countries? They don't see any ads?

Reply:
Note that in the legacy platform, ROW = amp-geo-group-eea OR amp-geo-no-group

Reply:
These are the CSS rules for the current platform:

```
.geo-amp-ad
{display:none}

.amp-geo-group-us .geo-amp-ad--us,
.amp-geo-group-au .geo-amp-ad--au,
.amp-geo-group-eea .geo-amp-ad--row,
.amp-geo-no-group .geo-amp-ad--row
{display:block}

.amp-geo-group-us .liveblog-geo-amp-ad--us,
.amp-geo-group-au .liveblog-geo-amp-ad--au,
.amp-geo-group-eea .liveblog-geo-amp-ad--row,
.amp-geo-no-group .liveblog-geo-amp-ad--row
{display:inline-block}
```

Reply:
@guardiancosta Should be able to test this with `#amp-geo=ZA` for instance, in dev mode.    
And also can see results in AdManager if you filter on `rp=dotcom-rendering` and `p=amp` and group by country.

Reply:
Fixed this issue here: https://github.com/guardian/dotcom-rendering/pull/528